### DISCUSSION TITLE
Do I have xtwist compensation on my SV06 plus

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#174] [Q&A]
I've read here : [https://github.com/bassamanator/Sovol-SV06-firmware/discussions/81](url)

So I understand the signs? but is it possible to print a first layer across the entire build plate to access this more definitively? What would you see?

Should this be enabled as it seems suggested by the discussion linked above. IE is there a down side if u don't have x twist? 

Did this get all finished up for the SV06 plus in the latest rendition of Bassam's set up

**Comment by bassamanator** [comment]
> is there a downside?

Well, while doing the `paper test` during axis twist compensation, if you do it poorly, yes, that would be a downside.

Axis twist is part of klipper official, so as long as you're up to date with klipper, you're good to go.

> but is it possible to print a first layer across the entire build plate to access this more definitively? What would you see?

Why not print a cube that covers your entire bed and is only 0.26mm in height? You're first layer should be uniform accross the entire build plate, meaning, bad everywhere mostly, good everywhere mostly, but not good on half the build plate *and* bad on the other half.
